src/client/components/body-portal/body-portal.tsx

```diff
@@ -32,7 +32,11 @@ export interface BodyPortalProps {
   isAboveAll?: boolean;
 }
 
-export class BodyPortal extends React.Component<BodyPortalProps, {}> {
+interface BodyPortalState {
+  isMounted: boolean;
```

Comment:
I would call that field `isAttached` and set it right after `document.body.appendChild`. 

I don't like keeping such state, but even docs say to do this ...

Reply:
👍